src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -188,7 +187,17 @@ public static String formatErrorMsg(String resource) {
             {"R_invalidEnclaveSessionFailed", "invalidate enclave session failed."},
             {"R_invalidEnclaveType", "Invalid enclave type {0}."},
             {"R_keystorePassword", "keystore password was incorrect"},
-            {"R_enclaveNotEnabled", "The statement triggers enclave computations"},
+            {"R_enclaveNotEnabled", "The statement triggers enclave computations"}, {"R_hasClosed", "has closed"},
+            {"R_hasBeenClosed", "has been closed"}, {"R_cannotOpenServer", "Cannot open server"},
+            {"R_failedToAuthenticate", "Failed to authenticate"},
+            {"R_signinTooManyTimes", "You've tried to sign in too many times with an incorrect user ID or password."},
+            {"R_toSigninAdd", "To sign into this application, the account must be added to"},
+            {"R_sslConnectionError",
```

Comment:
No need to define this error message here, you can get it from SQLServerResource.

Reply:
why is this resolved?